src/server/chat/discord-manager.js

```diff
@@ -49,6 +49,16 @@ const discordManager = function() {
 			}, console.error);
 
 			this.client.on("error", console.error, console.error);
+
+			this.client.on("guildBanAdd", function(guild, user) {
+				db.user.setBanState(true, user.id);
```

Comment:
I think it would be nice to log these ban/unban events for the server. Wouldn't hurt to be able to read them back, at least?

Reply:
Discord has an audit log which you can also check. Regardless, I'll still add them as it is a bit nicer for debugging in case anything goes wrong.